src/parkapi_sources/converters/pam_bw/converter.py

```diff
@@ -0,0 +1,77 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from datetime import datetime, timezone
+from typing import Any
+
+from openpyxl.cell import Cell
+from openpyxl.workbook.workbook import Workbook
+from validataclass.exceptions import ValidationError
+from validataclass.validators import DataclassValidator
+
+from parkapi_sources.converters.base_converter.push import XlsxConverter
+from parkapi_sources.converters.neckarsulm.models import NeckarsulmRowInput
+from parkapi_sources.exceptions import ImportParkingSiteException
+from parkapi_sources.models import SourceInfo, StaticParkingSiteInput
+
+
+class PamBwPushConverter(XlsxConverter):
+    neckarsulm_row_validator = DataclassValidator(NeckarsulmRowInput)
```

Comment:
Neckarsulm row validator is also here. I guess its a mistake?

Reply:
Yep, copy paste leftover. Removed.